library/ecdsa.c

```diff
@@ -321,6 +321,7 @@ int mbedtls_ecdsa_write_signature( mbedtls_ecdsa_context *ctx, mbedtls_md_type_t
 {
     int ret;
     mbedtls_mpi r, s;
+    const size_t ssize = MBEDTLS_ECDSA_MAX_SIG_LEN( ctx->grp.pbits );
```

Comment:
Please document the rules governing the size of the `sig` buffer.

* Here, add a comment explaining that we assume the buffer to be large enough.
* In `ecdsa.h`, modify the note about the size of the `sig` buffer to refer to the `MBEDTLS_ECDSA_MAX_SIG_LEN` macro instead of giving an English approximation of it.

Reply:
Done